pkg/nodemodel/wellknownassets/cni.go

```diff
@@ -34,14 +34,24 @@ import (
 // https://github.com/kubernetes/kubernetes/issues/30338
 
 const (
-	// CNI tarball for k8s >= 1.22
 	defaultCNIAssetAmd64K8s_22 = "https://storage.googleapis.com/k8s-artifacts-cni/release/v0.9.1/cni-plugins-linux-amd64-v0.9.1.tgz"
 	defaultCNIAssetArm64K8s_22 = "https://storage.googleapis.com/k8s-artifacts-cni/release/v0.9.1/cni-plugins-linux-arm64-v0.9.1.tgz"
 
-	// CNI tarball for k8s >= 1.27
 	defaultCNIAssetAmd64K8s_27 = "https://storage.googleapis.com/k8s-artifacts-cni/release/v1.2.0/cni-plugins-linux-amd64-v1.2.0.tgz"
 	defaultCNIAssetArm64K8s_27 = "https://storage.googleapis.com/k8s-artifacts-cni/release/v1.2.0/cni-plugins-linux-arm64-v1.2.0.tgz"
 
+	defaultCNIAssetAmd64K8s_29 = "https://storage.googleapis.com/k8s-artifacts-cni/release/v1.3.0/cni-plugins-linux-amd64-v1.3.0.tgz"
```

Comment:
We keep this is in WIP until we get some consensus about mirroring CNI to `artifacts.k8s.io`?

Reply:
Doesn't matter much, as this is for kOps 1.32, which is early alpha. For sure we will add mirrors for this.

Reply:
Also, please not that the top 3 use CNIs don't need those binaries, so even without mirrors, it should not be very demanding.

Reply:
Mirrors added 😄